ChatQnA/kubernetes/intel/cpu/xeon/manifest/chatqna.yaml

```diff
@@ -657,6 +629,8 @@ spec:
         app.kubernetes.io/name: data-prep
         app.kubernetes.io/instance: chatqna
     spec:
+      nodeSelector:
+        node-type: chatqna-ui
```

Comment:
please remove all the nodeSeletor

Reply:
Okay, this repository will be closed for now, and all test-related items will be removed.